### DISCUSSION TITLE
How to customize Iterable notification name in Android Settings?

### DISCUSSION THREAD

**Discussion by williamspete321** [Iterable/iterable-android-sdk#502] [Q&A] [answered]
Is there a way to customize the notification name of Iterable so that something more user friendly appears? For example, in the below photo of a build of our app, we see "iterable channel." Our users won't know what "iterable channel" refers too. So that our users don't disable this notification, it would be preferable if we could customize this name to something like "(Our App Name) Notifications."

![image001](https://user-images.githubusercontent.com/34874236/219753531-fd2bc07a-f12a-4e8b-8554-6c455fa6728c.png)

**Comment by Ayyanchira** [comment, marked as answer]
👋 Hi @williamspete321,

You should be able to provide a desired name to Iterable's notification channel by adding 
`<meta-data android:name="iterable_notification_channel_name" android:value="Custom Name"/>`
in the app's AndroidManifest file. 

[Iterable doc](https://support.iterable.com/hc/en-us/articles/115000331943-Setting-up-Android-Push-Notifications-#notification-channel-name) for more reference.

**Reply by williamspete321** [reply]
Thanks @Ayyanchira! 👍
